src/Chinchilla/Subscription.cs

```
@@ -65,6 +65,12 @@ public IWorkersController Workers
 
         public void Start()
         {
+            if (IsStarted)
+            {
+                logger.InfoFormat("Subscription already started: {0}", this);
```

@thingylab this should probably throw an exception, what do you think?